packages/mobile/src/screens/profile-screen/TracksTab.tsx

```diff
@@ -52,7 +50,6 @@ export const TracksTab = () => {
       leadingElementId={artist_pick_track_id}
       actions={tracksActions}
       lineup={lineup}
-      limit={track_count}
```

Comment:
oh interesting. is this limit needed for anything style-wise or something like that?

Reply:
nvm saw your comment

Reply:
We set the max items in the lineup on Track screen to 6, but idk why we would need it here. Removing this should be fine!